src/reset.css

```
@@ -0,0 +1,20 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
```

it's not reset
Move this value to the normalized file
also, move  this to the `html` selector

```
html {
  font-family: Roboto, Arial, sans-serif;
  font-weight: 500;
} 
```  
```suggestion
body {
  margin: 0;
}

p {
  margin: 0;
}

a {
  text-decoration: none;
}
```